README.md

```diff
@@ -49,6 +49,25 @@ const activity = require('@nearform/clinic-common/icons/activity.js')
 ```
 
 
+Basic style can be injected into the page by calling:
+`icons.injectStyle()`
```

Comment:
in flame we already use postcss-import so we could put the icons CSS in a file in clinic-common and do `@import "clinic-common/svg-icons.css"` or something in the tool's css. the other tools don't use postcss but we could add it there!

Reply:
Indeed!